compiler/lcalc/to_ocaml.ml

```diff
@@ -168,12 +181,14 @@ let rec format_typ (fmt : Format.formatter) (typ : Dcalc.Ast.typ Pos.marked) : u
   | TAny -> Format.fprintf fmt "_"
 
 let format_var (fmt : Format.formatter) (v : Var.t) : unit =
-  let lowercase_name = to_lowercase (to_ascii (Bindlib.name_of v)) in
+  let lowercase_name =
+    to_lowercase (to_ascii (Bindlib.name_of v ^ "_" ^ string_of_int (Bindlib.uid_of v)))
```

Comment:
Eventually, see if we really have to print the `uid`; I would prefer if we could just print the variable name since the scoping conventions match in OCaml.